test/e2e/load_test.go

```diff
@@ -38,91 +39,95 @@ var _ = Describe("Podman load", func() {
 	It("podman load input flag", func() {
 		outfile := filepath.Join(podmanTest.TempDir, "alpine.tar")
 
-		save := podmanTest.Podman([]string{"save", "-o", outfile, ALPINE})
+		images := podmanTest.PodmanNoCache([]string{"images"})
+		images.WaitWithDefaultTimeout()
+		fmt.Println(images.OutputToStringArray())
+
+		save := podmanTest.PodmanNoCache([]string{"save", "-o", outfile, ALPINE})
 		save.WaitWithDefaultTimeout()
 		Expect(save.ExitCode()).To(Equal(0))
 
-		rmi := podmanTest.Podman([]string{"rmi", ALPINE})
-		rmi.WaitWithDefaultTimeout()
-		Expect(rmi.ExitCode()).To(Equal(0))
+		//rmi := podmanTest.Podman([]string{"rmi", ALPINE})
+		//rmi.WaitWithDefaultTimeout()
+		//Expect(rmi.ExitCode()).To(Equal(0))
 
-		result := podmanTest.Podman([]string{"load", "-i", outfile})
+		result := podmanTest.PodmanNoCache([]string{"load", "-i", outfile})
```

Comment:
What, what? How does this work if we already have an image with the same ID present?

Reply:
well moreover, what is this even testing as I have it; lemme add the rmi's back in